sdk/python/packages/flet-cli/src/flet_cli/commands/build.py

```diff
@@ -235,331 +238,335 @@ def add_arguments(self, parser: argparse.ArgumentParser) -> None:
             "target_platform",
             type=str,
             choices=["macos", "linux", "windows", "web", "apk", "aab", "ipa"],
-            help="the type of a package or target platform to build",
+            help="The target platform or type of package to build",
         )
         parser.add_argument(
             "python_app_path",
             type=str,
             nargs="?",
             default=".",
-            help="path to a directory with a Python program",
+            help="Path to a directory with a Flet Python program",
         )
         parser.add_argument(
             "--arch",
             dest="target_arch",
             nargs="+",
             default=[],
-            help="package for specific architectures only. Used with Android and macOS "
-            "builds only.",
+            help="Build for specific CPU architectures "
+            "(used in macOS and Android builds only). Example: `--arch arm64 x64`",
         )
         parser.add_argument(
             "--exclude",
             dest="exclude",
             nargs="+",
             default=[],
-            help="exclude files and directories from a Python app package",
+            help="Files and/or directories to exclude from the package",
         )
         parser.add_argument(
             "-o",
             "--output",
             dest="output_dir",
-            help="where to put resulting executable or bundle (default is "
-            "<python_app_directory>/build/<target_platform>)",
             required=False,
+            help="Output directory for the final executable/bundle "
+            "(default: <python_app_path>/build/<target_platform>)",
         )
         parser.add_argument(
             "--clear-cache",
             dest="clear_cache",
             action="store_true",
             default=None,
-            help="clear build cache",
+            help="Remove any existing build cache before starting the build process.",
```

Comment:
It's not just I'm so picky, but some help messages end with `.`, and some don't :)
We need to choose the style.